### PR TITLE
Add `app_key` authentication to all swagger yaml files

### PR DESCRIPTION
**Pull request by maaktweluit**
- Updated `market-api.yml` to be compatible with the python client generator
- Copied the same security requirements to `payment` and `activity`

### CODE REVIEW

File: specs/activity-api.yaml [hunk 1]
```diff
@@ -383,6 +385,11 @@ paths:
               schema:
                 $ref: 'common.yaml#/schemas/ErrorMessage'
 components:
+  securitySchemes:
+    app_key:
+      type: apiKey
+      in: header
+      name: Authorization
```
Comment (tworec) [inline specs/activity-api.yaml:392]: we use https://swagger.io/docs/specification/authentication/bearer-authentication/, aren't we?

Comment (maaktweluit) [reply specs/activity-api.yaml]: Yes we are using that one.
There was a bug in the python client generator that required the above patch.
The bug has been fixed, so I updated this PR to use the schema as you linked.

File: specs/activity-api.yaml [hunk 2]
```diff
@@ -383,6 +385,10 @@ paths:
               schema:
                 $ref: 'common.yaml#/schemas/ErrorMessage'
 components:
+  securitySchemes:
```
Comment (tworec) [inline specs/activity-api.yaml:388]: What do you think of moving it to `common.yaml` as it is same for all the APIs?

Comment (maaktweluit) [reply specs/activity-api.yaml:392]: Good idea!

Tried to reference both the `security` and the `securitySchemes`, but only the latter worked.

Updated the PR.

**Pull request merged**